### PR TITLE
Keep 5 decimal places on converting mg/dL to mmol/L (to do otherwise implies greater accuracy than is available)

### PR DESCRIPTION
**Pull request by darinkrauss**
@jh-bate What it says above.

### CODE REVIEW

File: data/blood/glucose/glucose.go [hunk 1]
```diff
@@ -62,7 +63,9 @@ func NormalizeValueForUnits(value *float64, units *string) *float64 {
 	if value != nil && units != nil {
 		switch *units {
 		case MgdL, Mgdl:
-			return app.FloatAsPointer(*value / MmolLToMgdLConversionFactor)
+			intValue := int(*value/MmolLToMgdLConversionFactor*MmolLToMgdLPrecisionFactor + 0.5)
```
Comment (jh-bate) [inline data/blood/glucose/glucose.go]: the `0.5`?

Comment (darinkrauss) [inline data/blood/glucose/glucose.go]: It is just to force the integer value to round (rather than floor). Do you think it needs a constant?

Comment (jh-bate) [inline data/blood/glucose/glucose.go]: makes sense - maybe just a short comment?

Comment (darinkrauss) [inline data/blood/glucose/glucose.go]: Okay, will do.

**Comment by darinkrauss** [comment]
[<0:ux 5:dev 2:tst> [platform] Implement Jellyfish de-duplication algorithm in Platform for Medtronic support](https://trello.com/c/GExalbvy/50-0-ux-5-dev-2-tst-platform-implement-jellyfish-de-duplication-algorithm-in-platform-for-medtronic-support)

**Pull request merged**